rma/models/rma.py

```diff
@@ -1008,6 +1010,8 @@ def _prepare_refund(self, invoice_form, origin):
         """
         self.ensure_one()
         invoice_form.partner_id = self.partner_invoice_id
+        # Avoid set partner default value
+        invoice_form.invoice_payment_term_id = self.env["account.payment.term"]
```

Comment:
Use False instead

Reply:
I had tried to define False, but in a Form() it is not possible.